packages/theme-check-common/src/test/MockFileSystem.spec.ts

```diff
@@ -0,0 +1,71 @@
+import { beforeEach, describe, expect, it } from 'vitest';
+import { MockFileSystem } from './MockFileSystem';
+import { FileType } from '../AbstractFileSystem';
+
+describe('MockFileSystem', () => {
+  let fs: MockFileSystem;
```

Comment:
Idk why but testing a mock file system entirely made for testing with more tests is pretty funny to me 😄